docker/test/performance-comparison/config/config.d/user_files.xml

```diff
@@ -4,4 +4,7 @@
 
     <!-- Path to configuration file with users, access rights, profiles of settings, quotas. -->
     <users_config>users.xml</users_config>
-</yandex>
\ No newline at end of file
+
+    <!-- Path to directory where users created by SQL commands are stored. -->
+    <access_control_path>access/</access_control_path>
```

Comment:
Hm, does relative path work?

Reply:
Should work, but overall, this PR is still in "blind fix" state. I am waiting for CI results...